measure/quality/measure-quality

```diff
@@ -475,7 +475,7 @@ function get_coding_commandline_options() {
       if [ "$use_enctools" = "yes" ]; then
         options_qsv+="-look_ahead_depth 40 -extbrc 1 -b_strategy 1 -adaptive_i 1 -adaptive_b 0 -bf 7 -refs 4 -g 256 -idr_interval begin_only -strict -1 -async_depth 1"
```

Comment:
Oooops we seem to missed to enable AGOP for QSV...  This needs to be changed to -adaptive_b 0..

Reply:
Good notice. However, let's do this separately this time since this is the last PR to upstream our current state of the project. Let's first have status quo then improve.

Reply:
-adaptive_b 0  -->  -adaptive_b 1

Reply:
Wait, we have docs already exposed as `adaptive_b 1`. Ah, then we need to change **now**.

Reply:
fixed